gnuradio-runtime/include/gnuradio/tagged_stream_block.h

```diff
@@ -38,6 +39,79 @@ class GR_RUNTIME_API tagged_stream_block : public block
                         gr::io_signature::sptr output_signature,
                         const std::string& length_tag_key);
 
+    /*!
+     * \brief Given a [start,end), returns a vector of all tags in the range.
+     *
+     * \see block::get_tags_in_range
+     *
+     * Does the same as block::get_tags_in_range, but omits the length_tag_key-tagged tags
```

Comment:
I would suggest adding a comment here like:
```suggestion
     * Does the same as block::get_tags_in_range, but omits the length_tag_key-tagged tags
     *
     * This hides `block:get_tags_in_range()`, but does not overload it. When calling get_tags_in_range() from work(), this function will be called.
```

Reply:
it is an overload, just not one that is bendy at runtime :) but yeah, comments are good.

Reply:
(hm, not sure I was using the right words there. I think "overloading" in C++ is only applied to "operator"; but what we both seem to mean is "this is inheritance, with the function getting resolved to different things, depending on whether the type of the object handled is an instance of `block` or `tagged_stream_block`.